@xen-orchestra/lite/src/components/ui/UiStatusPill.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <UiIcon :icon class="icon" :class="props.state" />
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@/components/ui/icon/UiIcon.vue'
```

Comment:
You can use the `UiIcon` from `web-core`.
```suggestion
import UiIcon from '@core/components/icon/UiIcon.vue'
```